### PR TITLE
AGRC -> UGRC tags

### PR DESCRIPTION
**Pull request by stdavis**
Also a few misc things.

image

### CODE REVIEW

File: src/sweeper/sweepers/metadata.py [hunk 1]
```diff
@@ -17,7 +17,7 @@
 
 #: these constants were copied from https://github.com/agrc/agol-validator/blob/master/validate.py
```
Comment (steveoh) [inline src/sweeper/sweepers/metadata.py:18]: **question**: is this still true? Are they still synchronized?

**Review by steveoh — COMMENTED** [review]
What will happen with existing agrc tags on current items? It looks like this is only going to affect capitalization and not replacement?

**Comment by stdavis** [comment]
>What will happen with existing agrc tags on current items? It looks like this is only going to affect capitalization and not replacement?

It doesn't look like `AGRC` was ever a required tag. I could add `UGRC`. Should it be?

**Comment by steveoh** [comment]
> It doesn't look like `AGRC` was ever a required tag. I could add `UGRC`. Should it be?

https://gis.utah.gov/about/policy/metadata/#tags
> At a minimum, each SGID dataset’s tags should include the stewarding agency (ie: UGRC), “SGID”, and the appropriate category name.

File: src/sweeper/sweepers/metadata.py (hunk 1 above)
Comment (stdavis) [reply src/sweeper/sweepers/metadata.py]: They are now with the exception of `ugrc`. Auditor is missing that tag in main but it looks like [Jake is on top of it](https://github.com/agrc/auditor/pull/62/files#diff-77cebe02e1f55354178f02ed887d6123ae7e30527a2a243c17ae2dd9c1645021).

**Comment by stdavis** [comment]
> It might be difficult to know the stewarding agency to make this work. I wonder if that's why we didn't do this originally.

Yep, I bet that's the issue. I think that I could just write a script to crawl internal and replace `AGRC` with `UGRC` and then be done with it. What do you think?

**Review by steveoh — APPROVED** [review]

**Comment by stdavis** [comment]
I bumped the version to 1.2.1 and installed it on 57. I'll post a gist of the script to update the existing tags in slack for review before I run it.

**Pull request merged**